bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -341,4 +345,8 @@ private void writeToDisk(@NonNull String uuid,
                 .apply();
     }
 
+    private boolean openedFromDeeplink(Activity activity) {
```

Comment:
The `Activity` parameter here should have `@NonNull`. Also, for general organizational reasons the methods here are alphabetized, so this method should come before `queueDiskWritingIfNecessary`.